src/gpo/assets/en-US/PowerToys.adml

```diff
@@ -29,6 +30,7 @@
       <string id="SUPPORTED_POWERTOYS_0_84_0">PowerToys version 0.84.0 or later</string>
       <string id="SUPPORTED_POWERTOYS_0_85_0">PowerToys version 0.85.0 or later</string>
       <string id="SUPPORTED_POWERTOYS_0_86_0">PowerToys version 0.86.0 or later</string>
+      <string id="DEPRECATED_POWERTOYS_0_88_0">Deprecated on PowerToys version 0.88.0</string>
```

Comment:
better "from ... to ...". On older Powertoys Version it still works. Thoughts?

Thoughts?

Reply:
Hmmm. More text than required.

The field is called supported on and it is supported from `0.64` to `0.87`. So I think "PowerToys version 0.64 to PowerToys version 0.87" is the better because less text.

And for the string ID `SUPPORTED_POWERTOYS_0_64_0_TO_0_87_0` fits better within the logic of the other ones. Especially as the xml property and the ui element is called "Supported on".

Reply:
OK 🤔 Pushed another try.
What about this?
![image](https://github.com/user-attachments/assets/64e64bd3-508a-4b4c-bf1a-61cc70c5d780)
"From PowerToys version 0.64.0 until PowerToys version 0.87.1"

Reply:
WDYT @htcfreek ?

Reply:
should be good to go with

Reply:
Thank you for the review!